tests/phpunit/src/Gateway/PayplugPproTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Payplug\PayplugWoocommerce\Controller\PayplugGenericGateway;
+use WC_Order;
+
+class PayplugPproTest extends TestCase
```

Comment:
you're not testing the PPro gateway, you're testing the generic.
The name of the class should be changed